recipes/cdna_cupcake/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+$PYTHON setup.py build
+$PYTHON setup.py install
+
+cp annotation/*.py ${PREFIX}/bin/
+cp bacteria/*.py ${PREFIX}/bin/
+cp phasing/[a-zA-Z]*.py ${PREFIX}/bin/
+cp phasing/io/[a-zA-Z]*.py ${PREFIX}/bin/
+cp post_isoseq_cluster/*.py ${PREFIX}/bin/
+cp SequelQC/*.py ${PREFIX}/bin
```

Comment:
Its in general not good to copy everything in /bin. Is this really needed, are all binaries? Why is setup.py install not taking care of this?

Reply:
cDNA_Cupcake is a collection of various scripts, organized in different sub-directories. That's why I am copying all of these individual scripts into bin/. Only one set of scripts requires compilation and uses setup.py. If there is a better way of handling this, please let me know.